libvast/vast/detail/concepts.hpp

```diff
@@ -44,4 +59,31 @@ concept signed_integral = integral<T> && std::is_signed_v<T>;
 template <class T>
 concept floating_point = std::is_floating_point_v<T>;
 
+struct any_callable {
+  template <class... Ts>
+  void operator()(Ts&&...);
+};
+
+/// Inspectables
+template <class T>
+concept inspectable = requires(detail::any_callable& i, T& x) {
+  {inspect(i, x)};
+};
+
+/// push_back
```

Comment:
I think this comment is out-of-date or in the wrong place. I'd recommend just removing it.